src/get-arch.ts

```diff
@@ -0,0 +1,13 @@
+export default function getArch(arch: string): string {
+  if (arch === 'ia32') {
```

Comment:
Excellent ʕ◔ϖ◔ʔ!

Today, the GitHub Actions runner supports only `x64`, `arm`, and `arm64` excluding `x86` (32bit). It may not be necessary, I think.
https://github.com/actions/runner/releases/tag/v2.278.0

Reply:
Oh! Good to know. I just had taken a look on Hugo's release page.